src/compute_shader.rs

```diff
@@ -217,11 +219,18 @@ impl<S: ComputeShader> Default for ExtractedShaders<S> {
     }
 }
 
+/// Handle for the compute shader asset
+#[derive(Component)]
+pub struct ShaderHandle<S: Asset> {
+    /// The handle
+    pub handle: Handle<S>,
+}
```

Comment:
`Handle` doesn't implement `Component` anymore. Documentation could probably be improved.